webapp/src/language/en.json

```diff
@@ -86,7 +86,11 @@
     "community": "Community",
     "changelog": "Changelog",
     "github": "Github Repository",
-    "telegram": "Telegram Channel"
+    "telegram": "Telegram Channel",
+    "lacchain": "LACChain EOSIO Network Dashboard and Infrastructure Monitor",
+    "mainnet": "EOS Mainnet Network Dashboard and Infrastructure Monitor",
+    "lacchain>nodes": "LACChain EOSIO Nodes Status Monitor",
+    "mainnet>nodes": "EOS Mainnet Block Producer Nodes Status Monitor"
```

Comment:
can we add page titles for the remaining pages?

Reply:
sure, just give the titles list for rest of the pages

Reply:
done!